libraries/matchers.rb

```diff
@@ -111,6 +111,13 @@ def execute_jenkins_script(resource_name)
       resource_name)
   end
 
+  def execute_jenkins_scriptFile(resource_name)
+    ChefSpec::Matchers::ResourceMatcher.new(
+      :jenkins_scriptFile,
+      :execute,
+      resource_name)
+  end
```

Comment:
per above comment, this should be `execute_jenkins_script_file` and `:jenkins_script_file`.